holoviews/tests/teststreams.py

```diff
@@ -1230,6 +1270,10 @@ def test_selection_expr_stream_hist_invert_xaxis_yaxis(self):
 
     def test_selection_expr_stream_polygon_index_cols(self):
         # Create SelectionExpr on element
+        try:
+            import shapely
+        except:
+            raise SkipTest('Shapely required for polygon selection')
```

Comment:
Thanks, though this requires **either** shapely or spatialpandas.

Reply:
Addressed in f9370a69c